docs/architecture.md

```diff
@@ -1,12 +1,16 @@
 # Architecture
 
-The core of OSP-Magnum is best described as a library that provides a set of functions and data structures intended to support a space flight simulator. The user is responsible for implementing their own application on the top level, allowing for full control over every feature.
+The core of OSP-Magnum is a game engine, not a game.
```

Comment:
Game engine usually implies that it's a full pre-built application that can support games like Unity, which really isn't the case here.